### PR TITLE
Document Contribution

### PR DESCRIPTION
**Pull request by UnloadingGnat**
Completed Contribution overview.

### CODE REVIEW

**Comment by UnloadingGnat** [comment]
## Overview

Completed the contribution page of the docs, using contributing guidelines from https://github.com/huff-language/huff-rs/blob/main/README.md

## Checklist

- [x] Completed Contribution page for docs
- [x] Added EVM codes and huff-examples to resources

File: src/contribute/overview/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+All contributions are welcome! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+
+Contributing to Huff Language is not just limited to writing code. It can mean a wide array of things, such as participating in discussions in our [discord server](https://discord.gg/h8pkspwx) or reporting issues.
```
Comment (abigger87) [inline src/contribute/overview/README.md:15]: Let's add a blurb above that all contributions are welcome for all huff-language ecosystem repositories, with `huff-rs` being the primary repository for Huff Lang syntax + compilation.

File: src/resources/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ## Resources
 
-<insert useful links, resources, third party tools, etc, etc>
\ No newline at end of file
+[EVM Codes](https://evm.codes/) *An interactive reference to Ethereum Virtual Machine Opcodes*
+
+[huff examples](https://github.com/huff-language/huff-examples/blob/main/erc20/contracts/ERC20.huff) *A collection of vigorously tested, exemplary Huff contracts for demonstration purposes.
```
Comment (abigger87) [inline src/resources/README.md:5]: Change link to: `https://github.com/huff-language/huff-examples`

**Review by abigger87 — APPROVED** [review]
Minor nits, otherwise nice!

**Comment by UnloadingGnat** [comment]
> Minor nits, otherwise nice!

Thanks!

I added the blurb and fixed the link.

**Pull request merged**